samples/poc-subapp/xclap.js

```diff
@@ -2,7 +2,7 @@
  * Tell Electrode app archetype that you want to use ES6 syntax in your server code
  */
 
-process.env.SERVER_ES6 = true;
+// process.env.SERVER_ES6 = true;
```

Comment:
Since this is commented out, perhaps the above comment could be amended to indicate what the default value is?

Reply:
this is for the sample, our generator is generating this. 
the archetype now is default to support ES6 for server so we would update this in the generator.